appcontext/context.go

```diff
@@ -63,6 +63,8 @@ type Context struct {
 	RetryData []byte
 	// SecretProvider exposes the support for getting and storing secrets
 	SecretProvider security.SecretProvider
+	// ResponseContentType sets a custom response type
```

Comment:
Perhaps something like:
// ResponseContentType is used for holding custom response type for HTTP Trigger.

Reply:
This has been updated. You're now who I'm going to use for eloquent comments. ;-)